boards/v2/synopsys/nsim/Kconfig.nsim

```diff
@@ -0,0 +1,84 @@
+# DesignWare ARC nSIM simulated platform configuration
+
+# Copyright (c) 2016, 2019 Synopsys, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_NSIM
+	bool
+	select HAS_COVERAGE_SUPPORT
+	help
+	  The DesignWare ARC nSIM board is a virtual board based on
+	  the ARC nSIM simulator. It demonstrates the ARC core features
+	  and a console based on the ns16550 UART model.
+
+config BOARD_NSIM_NSIM_EM4
+	bool
+	select BOARD_NSIM
+
+config BOARD_NSIM_NSIM_EM7D_V22
+	bool
+	select BOARD_NSIM
+
+config BOARD_NSIM_NSIM_EM11D
+	bool
```

Comment:
```suggestion
```

please cleanup following lines, even though not commented directly.

Reply:
Fixed